plugins/by-name/blink/default.nix

```
@@ -0,0 +1,71 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "blink";
+  originalName = "blink.cmp";
+  package = "blink.cmp";
+
+  maintainers = [ lib.maintainers.balssh ];
+
+  description = ''
+    blink.cmp is a completion plugin with support for LSPs and external sources while updating on every keystroke with minimal overhead (0.5-4ms async). It achieves this by writing the fuzzy searching in SIMD to easily handle >20k items. It provides extensibility via hooks into the trigger, sources and rendering pipeline. Plenty of work has been put into making each stage of the pipeline as intelligent as possible, such as frecency and proximity bonus on fuzzy matching, and this work is on-going.
+  '';
+
+  settingsOptions = {
+    keymap =
+      defaultNullOpts.mkNullable (types.either types.attrs (types.enum [ false ]))
```

```suggestion
      defaultNullOpts.mkNullableWithRaw (with types; either (attrsOf anything) (enum [ false ]))
```

We need `WithRaw` here to support nixvim's `rawLua` type. We also should use `attrsOf anything` instead of `attrs` as the latter is deprecated.